### ISSUE TITLE
How to customize elpy-django-command and elpy-django-runserver for Windows

### ISSUE THREAD

**Issue by saltycraig** [jorgenschaefer/elpy#1310]
On Windows, even though the manage.py file is set as executable, I don't have permissions to set it to associate with my virtualenv python. So when I run ```M-x elpy-django-runserver``` instead of trying to run "c:/Users/craig/projects/foo/bar/manage.py runserver 127.0.0.1:8000" I'd like to modify the code/variable somewhere so that it adds 'python' before the manage.py like so:
"c:/Users/craig/projects/foo/bar/python manage.py runserver 127.0.0.1:8000". That way it avoids not having to have administrator rights to associate the manage.py file with virtualenv python I have setup,

**Comment by gopar** [comment]
Hey, Thanks for the report \o/

True, I didn't think about windows when I was creating this. I'll make a pr with the appropriate changes

**Comment by saltycraig** [comment]
Thanks very much! Anything else I notice I will submit issues.

**Comment by gopar** [comment]
@saltycraig question.

I'm assuming that the prefix of `python` command should only happen when using `manage.py` and not `django-admin.py`? Looks like some ppl do but not sure if this is still the case (Don't have windows to test) https://stackoverflow.com/q/2870262/2178312

**Comment by saltycraig** [comment]
Just tried this out, after activating a virtual env through cmd.exe (default terminal), I can use django-admin command with typical 'django-admin ...' So it appears yes, only when using manage.py command.

**Comment by gopar** [comment]
@saltycraig hey just a quick thought.
would doing something like this in the `.dir-locals.el` help?
```lisp
(eval . (setq elpy-django-command (concat "python " "/path/to/manage.py")))
```
Forgot that the `elpy-django-command` is customizable :P

If not, then I'll do the fix. Sorry, for being so behind :P

**Comment by saltycraig** [comment]
I've tried your elisp but emacs does not like that format, complaining about wrong type 'listp, setq'. I looked up .dir-locals.el in the manual and wrote this instead. I stuck in a .dir-locals.el file at the root of my project.

```
((nil . ((elpy-django-command . "python c:/Users/craig/projects/r/website/manage.py"))))
```

Now when I visit a file in the project after activating my virtual env, it will read the .dir-locals.el file fine and I can run ```M-x elpy-django-command``` and get back the expect "Command:" prompt with all the completions available, so good news there!
